setup.cfg

```diff
@@ -65,6 +65,14 @@ markers =
 max-line-length = 88
 exclude = build/*, dist/*, pip_tools.egg-info/*, piptools/_compat/*, .tox/*, .venv/*, .git/*, .eggs/*
 extend-ignore = E203  # E203 conflicts with PEP8; see https://github.com/psf/black#slices
+# flake8-pytest-style
```

Comment:
Let's add a separate line here to visually separate it from the built-in settings even more.
```suggestion

# flake8-pytest-style
```